tests/unit/cn/vblock_reader_test.c

```diff
@@ -257,7 +257,7 @@ MTF_DEFINE_UTEST_PRE(vblock_reader_test, t_detect_version1, pre)
 
     /* vbh_version is wrong, and should be detected in vbr_desc_read */
     omf_set_vbh_magic(&vbhdr, VBLOCK_HDR_MAGIC);
-    omf_set_vbh_version(&vbhdr, VBLOCK_HDR_VERSION1);
+    omf_set_vbh_version(&vbhdr, VBLOCK_HDR_VERSION + 1);
```

Comment:
the test case name no longer matches the test.  it is testing that future versions return EPROTO.  Maybe set vbh_version to 1 so that it tests version 1?  Or remove this test case and add a loop to `t_detect_bad_version` to test:  0, 1, N+1, -1, etc.

Reply:
Done